Classes/TMDebugConsole.m

```diff
@@ -186,27 +185,42 @@ - (void)setUpDateFormatter
 
 - (void)logMessage:(DDLogMessage *)logMessage
 {
+	logMessage = [self formatLogMessage:logMessage];
+	[self scheduleLogMessage:logMessage];
+}
+
+- (DDLogMessage *)formatLogMessage:(DDLogMessage *)logMessage
+{
+	if (formatter)
+	{
+		NSString *logMsg = [formatter formatLogMessage:logMessage];
+		if (logMsg != nil)
+		{
+			logMessage->logMsg = logMsg;
+		}
+		else
+		{
+			logMessage = nil;
+		}
+	}
+	return logMessage;
```

Comment:
Just a comment, I think this is confusing. Given the method definition, it looks like it is going to return a new `DDLogMessage` instance, however it is returning the same instance. In my opinion, the method should:
- Return a new instance
- Return `void` and notice that the param is an input/output argument.